app/packs/src/models/Element.js

```diff
@@ -119,6 +119,27 @@ export default class Element {
     return target;
   }
 
+  // Return true if the element has at least one analysis
+  analysesPresent() {
+    if (!this.container) { return false; }
+
+    const analysesContainer = this.container.children?.find((container) => container?.container_type === 'analyses');
+    if (!analysesContainer) { return false; }
+
+    const analysis = analysesContainer.children?.find((container) => container?.container_type === 'analysis');
+    return !!analysis;
+  }
```

Comment:
could that function be shorten to
```suggestion
analysesPresent() {
   return analyses().length > 0
} 
```

Reply:
Indeed.
But I think it proposed change could be faster when the element has heaps of analyses :) 
Also  the functions  defined above that one could also have re-used, but I think they   they need a rewrite, so did not do that here.